pkg/apis/application/v1alpha1/types.go

```diff
@@ -200,11 +200,10 @@ type ResourceDetails struct {
 
 // DeploymentInfo contains information relevant to an application deployment
 type DeploymentInfo struct {
-	Params                      []ComponentParameter `json:"params" protobuf:"bytes,1,name=params"`
-	Revision                    string               `json:"revision" protobuf:"bytes,2,opt,name=revision"`
-	ComponentParameterOverrides []ComponentParameter `json:"componentParameterOverrides,omitempty" protobuf:"bytes,3,opt,name=componentParameterOverrides"`
-	DeployedAt                  metav1.Time          `json:"deployedAt" protobuf:"bytes,4,opt,name=deployedAt"`
-	ID                          int64                `json:"id" protobuf:"bytes,5,opt,name=id"`
+	Revision                    string               `json:"revision" protobuf:"bytes,1,opt,name=revision"`
+	ComponentParameterOverrides []ComponentParameter `json:"componentParameterOverrides,omitempty" protobuf:"bytes,2,opt,name=componentParameterOverrides"`
+	DeployedAt                  metav1.Time          `json:"deployedAt" protobuf:"bytes,3,opt,name=deployedAt"`
+	ID                          int64                `json:"id" protobuf:"bytes,4,opt,name=id"`
```

Comment:
Same comment. Protobufs should never be renumbered or backwards compatibility is broken.